docs/com.bitwarden.organizations.md

```diff
@@ -0,0 +1,79 @@
+[Table of contents](README.md#table-of-contents)
+
+# Bitwarden organization
+
+The `com.bitwarden.organization` doctype is used to group
+[ciphers](./com.bitwarden.ciphers.md) in a single unit and/or to share them with
+other [bitwarden contacts](./com.bitwarden.contacts.md).
+
+## Attributes
+
+- `name`: {string} - the name of the organization
+- `members`: {object} - the members of the organization (see below)
+- `defaultCollection`: {object} - the collection of the organization (see below)
+- `cozyMetadata`: {object} - the [cozyMetadata](README.md#document-metadata)
+
+## Members
+
+An organization can be shared with a list of members.
+
+- `email`: {string} - the email of the member from [bitwarden contacts](./com.bitwarden.contacts.md)
+- `key`: {string} - the public key of the member from [bitwarden contacts](./com.bitwarden.contacts.md)
+- `name`: {string} - the name of the member
+- `owner`: {bool} - true if the member owns the organization
+- `status`: {int} - the status of the member (`0` = `invited`, `1` = `accepted`, `2` = `confirmed`)
+- `user_id`: {string} - the id of the member from [bitwarden contacts](./com.bitwarden.contacts.md)
+
+Note that the `name` and `key` attributes may be missing if the member status is still `invided` as those data are sent only when he accepts the first [Cozy sharing](./io.cozy.sharings.md) from organization's owner.
+
+## Collection
+
+A collection is a container for [ciphers](./com.bitwarden.ciphers.md).
+
+A [ciphers](./com.bitwarden.ciphers.md) can live alone or in a collection and a collection lives in an organization.
+
+By construction the Bitwarden protocol allows an organization to contain one or more collections.
+
+But at Cozy we defined that an organization must contains exactly one collection. So there is no need seperate those entities, and so there is no dedicated doctype for a collection.
+
+The collection entity exists to comply with the Bitwarden protocol.
+
+- `_id`: {string} - the id of the collection
+- `name`: {string} - the name of the collection, encrypted as a cipherString with AES. At Cozy, the collection's `name` is the same as organization's one
```

Comment:
I don't get why we have the encrypted name of the collection, while it is the same of the organization, which is saved in plaintext in the document, here `test`?

Reply:
This is an heritage from Bitwarden's protocol.

On Bitwarden's solution: an organization is just an "administrative" tool to regroup collections and users. Its name is not really important as the UI doesn't display it (it is shown only on the organization management page).
The organization can contain multiple collections, and those collections are what is displayed on the UI and can be part of how the users manage his passwords. I suppose this is why its name is encrypted.

On Cozy, this mechanism appears weird because we chose to have 1 organization = 1 collection and so we always have the same name for both, which makes the name's encryption useless. But we can (should?) set a random name for the organization if we want to. But saying that, i'm wondering if the name displayed on Bob's email is the clear name from organization 🤔

Reply:
From a security point of view, I don't think we should have both the encrypted and plaintext value, this might lead to KPA attacks: https://fr.wikipedia.org/wiki/Attaque_%C3%A0_texte_clair_connu
We should probably remove the plaintext and only rely on the encrypted collection name to display the name. Or, if the organization name is really required by bitwarden, duplicate the encrypted collection name?

Reply:
I think we should avoid using the same text for the collection name and the organization name (even if I'm not worried about this attack as modern crypto is robust to such attacks).

> But saying that, i'm wondering if the name displayed on Bob's email is the clear name from organization thinking

No, I think it comes that the sharing description (so, we probably have the same text for 3 things: sharing description, organization name, and collection name).

Reply:
> I think we should avoid using the same text for the collection name and the organization name

I removed that mention even if it is the case in the current implementation. So we may try to fix it without impacting the documentation.